control_plane/src/bin/neon_local.rs

```diff
@@ -1018,7 +1018,7 @@ fn cli() -> Command {
                 .about("Create a new blank timeline")
                 .arg(tenant_id_arg.clone())
                 .arg(branch_name_arg.clone())
-                .arg(region_id_arg.clone())
+                .arg(region_id_arg)
```

Comment:
This seems inconsistent with all other arguments to commands. and even region_id in other commands.

Reply:
same as above